liquidsoap/radio.liq

```diff
@@ -52,6 +52,19 @@ server.register("update",namespace="metadata",
                  usage="update title=foo,album=bar,..",
                  icy_update)
 
+def add_to_queue(v) =
+  args = string.split(separator=",",v)
+  track = list.hd(args)
+  cue_out = list.hd(list.rev(args))
+  annotate_line = "annotate:liq_fade_in=0,liq_fade_out=0,liq_cue_in=0,liq_cue_out=#{cue_out}:#{track}"
+  request.create(annotate_line)
```

Comment:
At line 60, char 2-31:
Warning 3: This expression should have type unit.
No output defined, nothing to do.